mailchimp-connector/backendScripts/actions/importNodes.js

```diff
@@ -0,0 +1,119 @@
+            session.getItem("/" + entity.id).remove();
+        }
+        let listNode = nodeUtil.createPath(rootNode, entity.id, contentType);
+        PropertyUtil.setProperty(listNode, "id", entity.id);
+        PropertyUtil.setProperty(listNode, "name", entity.name);
+        PropertyUtil.setProperty(listNode, "members", entity.stats.member_count);
+        PropertyUtil.setProperty(listNode, "permission_reminder", entity.permission_reminder);
+        PropertyUtil.setProperty(listNode, "email_type_option", entity.email_type_option);
+
+        let contactNode = nodeUtil.createPath(listNode, "contact", "mgnl:contentNode");
+        PropertyUtil.setProperty(contactNode, "address1", entity.contact.address1);
+        PropertyUtil.setProperty(contactNode, "company", entity.contact.company);
+        PropertyUtil.setProperty(contactNode, "city", entity.contact.city);
+        PropertyUtil.setProperty(contactNode, "country", entity.contact.country);
+        PropertyUtil.setProperty(contactNode, "zip", entity.contact.zip);
+        PropertyUtil.setProperty(contactNode, "state", entity.contact.state);
+
+        let campaignDefaultsNode = nodeUtil.createPath(listNode, "campaign_defaults", "mgnl:contentNode");
+        PropertyUtil.setProperty(campaignDefaultsNode, "from_name", entity.campaign_defaults.from_name);
+        PropertyUtil.setProperty(campaignDefaultsNode, "from_email", entity.campaign_defaults.from_email);
+        PropertyUtil.setProperty(campaignDefaultsNode, "subject", entity.campaign_defaults.subject);
+        PropertyUtil.setProperty(campaignDefaultsNode, "language", entity.campaign_defaults.language);
+
+        nodeTypesUtil.Activatable.update(listNode, "Import", true);
+    });
+}
+
+var ImportNodes = function () {
+    let utils = new Utils();
+    let MgnlContext = Java.type("info.magnolia.context.MgnlContext");
+    this.execute = function () {
+        let restClient;
+        let session = MgnlContext.getJCRSession(this.parameters.get("workspace"));
+        let rootNode = session.getRootNode();
+        let Notification = Java.type("com.vaadin.ui.Notification");
+
+        try {
+            restClient = utils.getRestClient("mailchimpRestClient");
+            let res = restClient.invoke(this.parameters.get("restCall"));
+            let body = JSON.parse(res.getEntity());
+            let bodyProperty = this.parameters.get("bodyProperty");
+
+            if (body.hasOwnProperty(bodyProperty)) {
+                let items = body[bodyProperty];
+                let contentType = this.parameters.get("contentModel");
+
+                if (contentType ==="mlchmp:campaign") {
+                    ImportCampaignItems(items, session, rootNode, contentType);
+                } else if (contentType === "mlchmp:list") {
+                    ImportListItems(items, session, rootNode, contentType);
+                } else {
+                    throw "Unrecognized content type " + contentType;
```

Comment:
You can use javascript interpolation with backticks `This is my message to the people: ${myMessageVariable}`